### ISSUE TITLE
airgap installation always uses the wrong path

### ISSUE THREAD

**Issue by Xaseron** [replicatedhq/kots#2818]
I try to use this guide https://docs.replicated.com/enterprise/installing-existing-cluster-airgapped
Something in the installation process tries to be smart and horribly fails.
pushing to registry:

```shell
kubectl kots admin-console push-images ~/kotsadm.tar.gz mycompany.de/sdc/gitpod \
  --registry-username REDACTED \
  --registry-password REDACTED
```
so far so good that worked.

trying to do the airgapped install:
```
kubectl kots install gitpod \
  --kotsadm-namespace gitpod \
  --kotsadm-registry mycompany.de/sdc \
  --registry-username REDACTED \
  --registry-password REDACTED
```
Cannot find image because it tries to pull `mycompany.de/sdc/minio:RELEASE.2022-01-08T03-11-54Z`
No i'm trying to be smart because the `/gitpod` is missing in the path so i try to install it that way:
```
kubectl kots install gitpod \
  --kotsadm-namespace gitpod \
  --kotsadm-registry mycompany.de/sdc/gitpod \
  --registry-username REDACTED \
  --registry-password REDACTED
```

It still fails but now i tries to pull `mycompany.de/sdc/gitpod/gitpod/minio:RELEASE.2022-01-08T03-11-54Z`
WTF what is wrong with the install script?

To workaround this i have to push the images again with
```shell
kubectl kots admin-console push-images ~/kotsadm.tar.gz mycompany.de/sdc/gitpod/gitpod \
  --registry-username REDACTED \
  --registry-password REDACTED
```

**Comment by cbodonnell** [comment]
Hi @Xaseron, this looks to be a bug with the registry namespace based on the output you shared.

In the meantime, can you try working around this issue by omitting the `--kotsadm-namespace` flag and specifying `--kotsadm-registry` as `mycompany.de/sdc/gitpod`?

Thanks for bring this to our attention.

**Comment by cbodonnell** [comment]
@Xaseron, this issue was fixed in KOTS 1.86.1 (see release notes [here](https://docs.replicated.com/release-notes/rn-app-manager#bug-fixes-1-86-1)).
